wai-aria/heading/invalid-nesting.html

```diff
@@ -0,0 +1,216 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Parity tests for invalidly nested heading elements</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+
+<body>
+
+  <h1>Parity tests for invalidly nested heading elements</h1>
+
+  <h2>Accessible names</h2>
+  <hr>
+
+  <h1
+    class="ex-label"
+    data-testname="Invalid parent h1 has empty accname"
```

Comment:
I'm a fan of more verbose test names... they should be specific enough that the following is true:
- there's no chance of name conflict with another test.
- we don't have to update the name with a more specific qualifier when another test is added (this breaks the history log)
- ideally, any viewer looking at the WPT.fyi results will know what the test does... I don't think that's the case for most of these. 

As an example: "h1 with h2 descendant should not inherit label from the text contents of the contained h2"